stories/components/GuideCard.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { withKnobs, text, number, boolean } from '@storybook/addon-knobs';
+import StoryItem from '../styleguide/StoryItem';
+import GuideCard from '../../lib/GuideCard';
+
+const stories = storiesOf('Components', module);
```

Comment:
In line with recent chats newer or re-visited stories should be placed in the components they interact with. So this could live in `lib/GuideCard/stories/GuideCardStory.js`.